static/sass/_patterns_media-object--large.scss

```diff
@@ -0,0 +1,12 @@
+@mixin p-media-object--large {
+  .p-media-object__image--large {
+    @extend .p-media-object__image;
+    flex-shrink: 0;
+    height: 6rem;
+    width: 6rem;
+  }
+
+  .p-media-object__content--large {
+    margin-top: 1rem;
```

Comment:
This in theory would not be needed if we had more control over spacing.

Default Vanilla spacing between headings/paragraphs is too big, default Media object spacing (from `.p-media-object__content` is too small).